shell/platform/tizen/channels/key_event_channel.h

```diff
@@ -7,20 +7,26 @@
 
 #include <Ecore_Input.h>
 
+#include <memory>
```

Comment:
Why added this? Is there any special reason?

Reply:
Because `std::unique_ptr` is used by the file and it's defined in `<memory>`.

Reply:
Huh? But why can I build? Why didn't it have a problem before?

Reply:
Because the compiler adds it automatically? You can try removing all the standard headers from this and other files and the code will still build well. However I think explicitly specifying the required headers makes the code more robust.

Reply:
> Because the compiler adds it automatically? 

Really? :scream: I didn't know at all. Technology has advanced a lot. :smile: 
I totally agree with your intentions.

Reply:
Maybe the standard headers are already included by dependent headers like dlog.h.